src/app/catalogue-search/profile-filters/profile-filters.component.html

```diff
@@ -0,0 +1,53 @@
+<!--
+Copyright 2020-2023 University of Oxford and NHS England
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+<div class="mdm-profile-filters">
+  <div class="mdm-profile-filters__header">
+    <h3>MetaData Filters</h3>
```

Comment:
Rename the heading to "Profile Filters" to make it consistent with the main Search page.